src/main/resources/staff/templates/document/waterMarkedLegalSealedClaim.html

```diff
@@ -0,0 +1,547 @@
+<html>
+<head>
```

Comment:
NO, this is not the way to do it.

Add a variable to the content provider and then check for the variable